UserDetails/forms.py

```diff
@@ -59,8 +59,9 @@ class RegisterAssociationLinks(forms.Form):
     # Try except is needed in case Associations is not yet a created table
     # (this line is reached in a migrate command, which fails)
     try:
-        associations = forms.MultipleChoiceField(choices=[(a.pk, a.name) for a in Association.objects.all()],
-                                             help_text='At which associations are you active?')
+        associations = forms.MultipleChoiceField(
+            choices=[(a.pk, a.name) for a in Association.objects.filter(is_choosable=True)],
+            help_text='At which associations are you active?')
     except:
```

Comment:
You should never catch all exceptions, only the specific exception that you want to catch, otherwise it could hide problems. I don't know which exception needs to be catched, but if it's `DatabaseError`, write `except DatabaseError:` instead of `except:`.

Reply:
True. Moet even achterhalen welke error het was.